cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/meta/RemotePrivilegesHandler.java

```diff
@@ -78,9 +85,9 @@ public void grant(HttpRequest request, HttpResponder responder) throws Exception
     EntityId entityId = deserializeNext(arguments);
     Principal principal = deserializeNext(arguments);
     Set<Action> actions = deserializeNext(arguments, SET_OF_ACTIONS);
-    LOG.trace("Granting {} on {} to {}", actions, entityId, principal);
-    authorizer.grant(entityId, principal, actions);
-    LOG.debug("Granted {} on {} to {} successfully", actions, entityId, principal);
+    LOG.debug("Granting {} on {} to {}", actions, entityId, principal);
+    privilegesManager.grant(entityId, principal, actions);
+    LOG.info("Granted {} on {} to {} successfully", actions, entityId, principal);
```

Comment:
I think someone getting a new privilege or losing a privilege is important information for the user to know. Also, this should not happen very frequently. Right now it will only happen on entity creation, deletion or on manual grants and revokes.